.github/workflows/cmake.yml

```diff
@@ -28,13 +34,16 @@ jobs:
     - name: Setup
       run: |
         sudo apt update
-        sudo apt install -y libboost-all-dev
+        sudo apt install -y libboost-all-dev ninja-build ccache
+    - name: ccache
+      uses: hendrikmuhs/ccache-action@v1.2.2
+      with:
+        key: cmake-linux-nonstandard-ubuntu-latest
+        max-size: "2000M"
     - name: Compile
-      env:
-        CXXFLAGS: -O2
       run: |
-        cmake --preset linux-ci-build-with-nonstandard-options
-        cmake --build --preset linux-ci-build-with-nonstandard-options -j2
+        cmake --preset linux-ci-build-with-nonstandard-options -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
```

Comment:
`CMAKE_CXX_COMPILER_LAUNCHER` should be added to the `linux-ci-build-with-nonstandard-options` preset, as this preset is only used for CI builds and should include everything required for it.

Reply:
Thanks, that makes sense. We've just pushed that.